homeassistant/components/qnap/translations/en.json

```diff
@@ -0,0 +1,24 @@
+{
+  "config": {
+    "step": {
+      "user": {
```

Comment:
`en.json` file is not necessary, it will be automatically created after merging.

Reply:
OK I delete the folder.